app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
```

Comment:
Initialization function always returns `None`, it's unnecessary to define that for functions that returns `None`
```suggestion
    def __init__(self, name: str):
```